Will Heikoff Documentation

```diff
@@ -102,6 +102,19 @@ Issue re-opening Spyder; error stating that “Python has stopped working”; mo
 Tried reinstalling python in the Windows’ Anaconda Prompt window using the “conda install –c conda-forge python” function
 	Successful installation, yet still cannot open Spyder program
 Troubleshooting methods to reopen spyder
+
+7/5/2018
+Due to Spyder unable to able to open was unable to make necessary changes to Code relating to the movement of the 3-stage program 
+	folder by kbuzard to the kfunctions folder
+Troubleshooting to find the block level data set for the 3-stage programs by importing different data sets into the ArcMap file ‘Base’
+	found in the Base Data Folder (G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data) and searching for data set 
+	attributes which exhibit defined blocks in the data
+Began searching in the Materials for Block Level Analysis folder (G:\MAX-Filer\Collab\Labs
+	kbuzard-S18\Admin\Materials for Block Level Analysis); not successful
+Continuing search in the Base Data Folder (G:\MAX-Filer\Collab\Labs-kbuzard-
+	S18\Admin\CARR\California\Base Data)
```

Comment:
I think we'll have the most luck in the Tony Smith 2014 folder. You may have to read through the documentation file to figure out what's going on there--it is LONG and complicated, but it is thorough and should be very helpful.

Meanwhile, I've been troubleshooting the python issues all day and I got it to work on my machine just a bit ago. I've been able to get most of the 2-stage program to work. So I'm going to try to write up what I've done and I'll send it to you so you can try it on your machine.

Reply:
There's a text document at G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Patents\NE Good\environment_instructions.txt that I hope should get you up and running in python.

This is the content:

## Create python environment

1. Open Anaconda Prompt

2. Submit the following statement

```
conda create -n labs python=3.6 spyder pyshp scipy cython geopandas

```

3. Type "y" when it asks whether to proceed.

4. Type 
```
conda activate labs
```
to activate the environment.

5. Open K-function Local_Scipy.py from Spyder (labs). This will be a new instance of Sypder that is available that has pyshp (for shapefile), scipy and geopandas already. Geopandas seems to need `cython' to work.